src/utils/app/folders.ts

```diff
@@ -204,14 +204,25 @@ export const getPathToFolderById = (
 };
 
 export const getFilteredFolders = (
-  folders: FolderInterface[],
+  allFolders: FolderInterface[],
```

Comment:
I understand that this is not your changes, but could you please also change this 6 params to object - it will be more easier to pass arguments inside and you will not have issues with order and it will be more visible what params passed